.github/workflows/windows-tests.yaml

```diff
@@ -0,0 +1,65 @@
+name: Windows Tests
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  tests:
+    env:
+      ENV_NAME: ${{ matrix.env_name }}
+      PYTHON: ${{ matrix.python-version }}
+      OS: windows-latest
+    name: Windows Testing
+    runs-on: windows-latest
+    defaults:
+      run:
+        # Adding -l {0} helps ensure conda can be found properly.
+        shell: bash -l {0}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.10", "3.11", "3.12"]
+        env_name: [pyuvsim_tests_windows]
```

Comment:
Unless there are differences beyond just the os and env_name, I think this could be added to the existing testsuite.yaml workflow using the `include` options, similar to what we did for mpich. That would make it easier to maintain.

Reply:
The one difference is that for windows we don't use `-n auto` because it can result in weird filesystem errors.

Reply:
Ah yeah, sure!

Reply:
Thanks!